diff --git a/node_modules/handlebars/lib/index.js b/node_modules/handlebars/lib/index.js
index 7a239a5..2a8d4bb 100644
--- a/node_modules/handlebars/lib/index.js
+++ b/node_modules/handlebars/lib/index.js
@@ -29,11 +29,4 @@
 
 exports.create = create;
 
-var _create = create,
-  create = _create === undefined ? function () {} : _create;
-
-/* eslint-disable no-undef */
-if (typeof require !== 'undefined' && require.extensions) {
-  require.extensions['.handlebars'] = require.extensions['.hbs'] = function (module, filename) {
-    var fs = require('fs');
-    var templateString = fs.readFileSync(filename, 'utf8');
-    module.exports = create().compile(templateString);
-  };
-}
-/* eslint-disable no-undef */
\ No newline at end of file
+var _create = create, create = _create === undefined ? function () {} : _create;
